ten_thousand_feet.gemspec

```diff
@@ -4,26 +4,27 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'ten_thousand_feet/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "ten_thousand_feet"
+  spec.name          = 'ten_thousand_feet'
   spec.version       = TenThousandFeet::VERSION
-  spec.authors       = ["Kyle Suss", "Conor Livingston"]
-  spec.email         = ["rubygems@spindance.com"]
-  spec.description   = %q{The 10,000ft Ruby API Client}
-  spec.summary       = %q{Provides Ruby wrappers for the 10,000ft API.}
-  spec.homepage      = "https://github.com/spindance/ten_thousand_feet"
-  spec.license       = "MIT"
+  spec.authors       = ['Kyle Suss', 'Conor Livingston']
+  spec.email         = ['rubygems@spindance.com']
+  spec.description   = 'The 10,000ft Ruby API Client'
+  spec.summary       = 'Provides Ruby wrappers for the 10,000ft API.'
+  spec.homepage      = 'https://github.com/spindance/ten_thousand_feet'
+  spec.license       = 'MIT'
 
-  spec.files         = `git ls-files`.split($/)
+  spec.files         = `git ls-files`.split($RS)
```

Comment:
It think it's just a coincidence that ``git ls-files`.split($RS)` works in this context. The actual value for $RS in this context is `nil` because the English module hasn't been loaded.

Nil just happens to split it the way we want, but we want to split on whatever newline the host OS uses.
```
2.2.2 :006 > "line\nline\nline".split(nil)
 => ["line", "line", "line"]
```